cmd/loom/sendtx.go

```diff
@@ -50,16 +53,48 @@ func newMigrationCommand() *cobra.Command {
 			if callerChainID == "" {
 				callerChainID = cli.TxFlags.ChainID
 			}
-			return migrationTx(Id, cli.TxFlags.PrivFile, cli.TxFlags.Algo, callerChainID)
+			inputBytes, err := parseInputParameters(Id, inputFile)
+			if err != nil {
+				return err
+			}
+			return migrationTx(Id, inputBytes, cli.TxFlags.PrivFile, cli.TxFlags.Algo, callerChainID)
 		},
 	}
 	migrationCmd.Flags().Uint32Var(&Id, "id", 0, "migration ID")
+	migrationCmd.Flags().StringVarP(&inputFile, "inputParam", "i", "", "protocol buffers json file")
 	migrationCmd.Flags().StringVarP(&cli.TxFlags.PrivFile, "key", "k", "", "private key file")
 	setChainFlags(migrationCmd.Flags())
 	return migrationCmd
 }
 
-func migrationTx(migrationId uint32, privFile, algo, callerChainID string) error {
+//Parse Migration Input Parameters depending upon MigrationId
+func parseInputParameters(migrationId uint32, inputFile string) ([]byte, error) {
+	var inputBytes []byte
+	switch migrationId {
+	case 1:
```

Comment:
Should be: 

```
case 2:
default:
```

No advantage of having case 1 separately.

Reply:
Done.